tasks.c

```diff
@@ -3426,7 +3429,8 @@ void vTaskMissedYield( void )
  * void prvIdleTask( void *pvParameters );
  *
  */
-static portTASK_FUNCTION( prvIdleTask, pvParameters )
+
+portTASK_FUNCTION( prvIdleTask, pvParameters )
```

Comment:
Note code coverage for this line is not expected.  There is no change except to remove the `static` from the `function definition` because it is already defined in the `function declaration`.  Please waive.